src/eventListener.ts

```diff
@@ -1,9 +1,10 @@
 /**
- * @param {Array|Element} element
- * @param {Array|string} event
+ * @param {Array|HTMLElement} element
+ * @param {string} event
  * @param {Function} callback
  */
-function addEventListener (element, event, callback) {
+
+function addEventListener (element: Array<HTMLElement>|HTMLElement, event:string, callback: () => void) {
```

Comment:
Are you sure the callback part is correct? Can you explain it? Shouldn't it be callback: Function?

I guess you look this up somewhere so I probably just don't know about it, maybe you can point me to the resource?

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

And it says to put parameters as well if you know what the callback will be, but it appears that the function is not always the same.